src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/turbines/GT_MTE_LargeTurbine_Plasma.java

```diff
@@ -129,8 +129,15 @@ public boolean checkRecipeGeneric(ItemStack[] aItemInputs, FluidStack[] aFluidIn
                                     Float.MIN_NORMAL,
                                     ((GT_MetaGenerated_Tool) aStack.getItem()).getToolStats(aStack).getSpeedMultiplier()
                                             * GT_MetaGenerated_Tool.getPrimaryMaterial(aStack).mToolSpeed
-                                            * 50))
-                            * getSpeedMultiplier();
+                                            * 50));
+
+                        //Calculate total EU/t (as shown on turbine tooltip (Fast mode doesn't affect))
+                    double aEUPerTurbine = aTotalOptimalFlow
+                                    * 40 * 0.0105
```

Comment:
maybe it's classical info if you toy with plasma gen calculus every day, but from where the `40*0.0105` comes from?

Reply:
The fuel value (as shown on the tooltip) can be seen [here](https://github.com/GTNewHorizons/GT5-Unofficial/blob/f2648a0eee49982e9fddbf1d0367530e3861526c/src/main/java/gregtech/api/items/GT_MetaGenerated_Tool.java#L513), I just removed the division and absorbed the factor of 50 from the above line, so 2000 -> 40.

Reply:
and for the `0.0105` ? tuning value for the nerfs?

Reply:
Magic value extracted from the value on tooltip, I just removed the divison by 100.